src/types/ErrorMessage.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorMessage {
+  NONE = '',
+  TITLE = 'Title can\'t be empty',
+  LOAD = 'Unable to load a todo',
+  ADD = 'Unable to add a todo',
+  DELETE = 'Unable to delete a todo',
+  UPDATE = 'Unable to apdate a todo',
```

Comment:
There is a typo in the `UPDATE` error message. It should be 'Unable to update a todo' instead of 'Unable to apdate a todo'. 🧐